tests/Unit/Core/Form/IdentifiableObject/DataFormatter/BulkCombinationFormDataFormatterTest.php

```diff
@@ -208,5 +208,45 @@ public function getDataToFormat(): iterable
                 ],
             ],
         ];
+
+        yield 'images data' => [
+            [
+                'images' => [
+                    'images' => [1, 3, 5],
```

Comment:
I'm surprised, from the Formatter code I would expect this field to be only present when `disabling_switch_images` is true no? In this case, the `images` data should be simply ignored no?